nengo_gui/static/components/netgraph.js

```diff
@@ -70,6 +70,17 @@ Nengo.NetGraph = function(parent, args) {
         }
     });
 
+    Object.defineProperty(this, 'kept_time', {
+        get: function() {
+            return Nengo.config.kept_time;
+        },
+        set: function(val) {
+            if (val === this.kept_time) { return; }
+            Nengo.config.kept_time = val;
+            sim.time_slider.update_kept_time(val);
+        }
+    });
+
```

Comment:
Why is this in Nego.NetGraph?  It isn't used by any of the NetGraph components...